core/controllers/PhotoController.js

```diff
@@ -0,0 +1,15 @@
+import Photo from '../models/Photo.js';
+
+class PhotoController {
+  async getPhotosByAuthor(req, res, next) {}
+  async getPhotosPublicByAuthor(req, res, next) {}
+  async getLikesById(req, res, next) {}
+  async getCountByAuthor(req, res, next) {}
```

Comment:
The PhotoSchema already has `count` attr, should we have this route?

Reply:
router.get('/count/:user-id/', photoController.getCountByAuthor); Dùng để hiển thị tổng số lượng photo.